### PR TITLE
Product repository

### PR DESCRIPTION
**Pull request by mostafaism1**

### CODE REVIEW

File: src/main/java/com/example/onlinestore/model/Product.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import java.awt.*;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Entity
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int Id;
```
Comment (Jezorko) [inline src/main/java/com/example/onlinestore/model/Product.java]: [nitpick] instance field names should start with lowercase:
```suggestion
    private int id;
```

See [naming convention guide](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) (:

File: src/main/java/com/example/onlinestore/repository/ProductRepository.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.example.onlinestore.repository;
+
+import com.example.onlinestore.model.Product;
+import org.springframework.data.repository.CrudRepository;
+
+public interface ProductRepository extends CrudRepository<Product, Integer> {
```
Comment (Jezorko) [inline src/main/java/com/example/onlinestore/repository/ProductRepository.java]: Might be worth extending `JpaRepository` instead: https://stackoverflow.com/questions/14014086/what-is-difference-between-crudrepository-and-jparepository-interfaces-in-spring

**Review by Jezorko — APPROVED** [review]

File: src/main/java/com/example/onlinestore/repository/ProductRepository.java (hunk 2 above)
Comment (mostafaism1) [reply src/main/java/com/example/onlinestore/repository/ProductRepository.java]:
```suggestion
public interface ProductRepository extends JpaRepository<Product, Integer> {
```

File: src/main/java/com/example/onlinestore/repository/ProductRepository.java [hunk 3]
```diff
@@ -3,5 +3,5 @@
 import com.example.onlinestore.model.Product;
 import org.springframework.data.repository.CrudRepository;
```
Comment (Jezorko) [inline src/main/java/com/example/onlinestore/repository/ProductRepository.java]: You also need to change the import here (;

**Pull request merged**